package.json

```diff
@@ -7,7 +7,7 @@
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
     "lint": "vue-cli-service lint",
-    "codecov": "codecov -f coverage/coverage-final.json",
+    "codecov": "codecov -f coverage/coverage-final.json && codecov -f coverage/jest/coverage-final-json",
```

Comment:
Is there a typo in the jest cov file ?
`coverage-final-json` or `coverage-final.json` ?

Reply:
yes, there is. and a few more errors that I am dealing with